src/test/java/net/masterthought/cucumber/generators/helpers/TableRowAssertion.java

```diff
@@ -0,0 +1,48 @@
+package net.masterthought.cucumber.generators.helpers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.commons.lang.StringUtils;
+
+public class TableRowAssertion extends ReportAssertion {
+
+    public LinkAssertion getReportLink() {
+        return super.getLink();
+    }
+
+    WebAssertion[] getCells() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/2417958554/issues/source?bid=3059308&fileBranchId=3237472#l13)